src/main/resources/db/migration/V1.4.0__Add__categories.sql

```diff
@@ -0,0 +1,16 @@
+create table Category
```

Comment:
Since a V1.4.0 file was added in https://github.com/tarent/t-learn/pull/77 this file should now be V1.5.0 :)

Reply:
True. Done.